### ISSUE TITLE
Extraction of useful formatting functionality

### ISSUE THREAD

**Issue by scalvert** [dequelabs/react-axe#180]
I work on [a project](https://github.com/ember-a11y/ember-a11y-testing) that utilizes axe-core under the covers to provide auditing of violations for accessibility. This project previously had a feature that output violations to the console in a similar manner to `react-axe`. In reading the code in this repo, there's a number of very useful, and largely generic, utilities that if extracted could be broadly useful, regardless of the underlying framework. 

My question is, would the maintainers of this repository be interested in potentially extracting those useful pieces, potentially publishing them as a separate package (`axe-utils` or somesuchthing)? I'd be happy to do the work to extract the pieces!

**Comment by straker** [comment]
Thanks for the suggestion. What utilities / useful pieces were you thinking about?

**Comment by straker** [comment]
We have moved this package to be under our [axe-core-npm](https://github.com/dequelabs/axe-core-npm/) monorepo for easier management. As a result, this repository will be deprecated in the coming weeks. 

If you have ideas on which utilities or pieces you would find useful outside of react-axe, please feel free to open an issue in the new repository.
